mne/viz/_3d.py

```diff
@@ -836,8 +853,9 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
     skull_alpha = dict()
     skull_colors = dict()
     hemi_val = 0.5
+    max_alpha = 1.0 if len(other_picks['seeg']) == 0 else 0.75
     if src is None or (brain and any(s['type'] == 'surf' for s in src)):
-        hemi_val = 1.
+        hemi_val = max_alpha
     alphas = (4 - np.arange(len(skull) + 1)) * (0.5 / 4.)
```

Comment:
I would adjust this line to its linspace equivalent:
```
np.linspace(1. / 2., 0, 5)[:len(skull) + 1]
```
And then make the appropriate adjustment in this PR, namely to swap out the 1 to obtain:
```
np.linspace(max_alpha / 2., 0, 5)[:len(skull) + 1]
```
In other words, all defaults should get less opaque, not just the outermost one (scalp).

But if you thought about this and tried it already and it doesn't look as good that's fine! But it would still be nice to change to the `np.linspace` line now that I figured out what the heck it's doing. It's much more readable...

Reply:
great!